components/style/properties/longhand/box.mako.rs

```diff
@@ -2093,14 +2096,21 @@ ${helpers.single_keyword("transform-style",
     bitflags! {
         #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
         pub flags SpecifiedValue: u8 {
-            const SIZE = 0x01,
+            % if product != "gecko":
+                const SIZE = 0x01,
+            % endif
             const LAYOUT = 0x02,
             const STYLE = 0x04,
             const PAINT = 0x08,
-            const STRICT = 0x10,
-            const STRICT_BITS = SIZE.bits | LAYOUT.bits | STYLE.bits | PAINT.bits,
-            const CONTENT = 0x20,
-            const CONTENT_BITS = LAYOUT.bits | STYLE.bits | PAINT.bits,
+            % if product == "gecko":
+                const STRICT = 0x10,
```

Comment:
The strict definition is the same for both, so let's move this out.

Also, this feature is gecko-only, so I'd just remove the product checks and remove `contains: content`.